src/components/Modals/Send/Form.tsx

```diff
@@ -67,9 +69,20 @@ export const Form: React.FC<SendFormProps> = ({ initialAssetId, input = '', acco
       amountCryptoPrecision: '',
       fiatAmount: '',
       fiatSymbol: selectedCurrency,
+      txHash: '',
     },
   })
 
+  const handleSubmit = useCallback(
+    async (data: SendInput) => {
+      const txHash = await handleFormSend(data)
+      if (!txHash) return
```

Comment:
https://github.com/shapeshift/web/pull/7698/files#r1750269181

If there is an error, we will have an empty string but redirect to the status page, is this wanted?

Reply:
Not sure what you mean @NeOMakinG:

- if there is an *on-chain* error (but the Tx is broadcasted fine), this will be detected at status step, which is an improvement over the previous implementation, as we would be optimistic and not detect on-chain errrored Txs.
- if there is a *broadcast* error, we won't have a txHash, hence the modal will be kept in its current state, and will fire a toast in `handleFormSend()` same as previously. We do *not* route to status in this case, since we early return.

Reply:
It's only true because the returned empty string has been removed, if there was a broadcast error previously, `handleFormSend` would have returned an empty string and so the early return wouldn't work

But it's fine now that `handleFormSend` returns undefined 👍